src/main/java/org/folio/circulation/domain/Loan.java

```diff
@@ -95,6 +95,14 @@ public JsonObject asJson() {
     return representation.copy();
   }
 
+  public JsonObject asJsonWithUserAndProxy() {
+    JsonObject json = asJson();
+    write(json, "item", JsonObject.mapFrom(item));
```

Comment:
Please make sure here and in similar places that following `JsonObject.mapFrom(item)` results into a proper structure.